.github/workflows/test-pr.yml

```diff
@@ -116,14 +116,45 @@ jobs:
             docker exec -u github-user kontrol-ci-integration-${GITHUB_SHA} /bin/bash -c 'CXX=clang++-14 poetry run kdist --verbose build -j`nproc` evm-semantics.haskell kontrol.foundry'
       - name: 'Run integration tests'
         run: |
-          TEST_ARGS='--numprocesses=8 -vv'
+          TEST_ARGS='--numprocesses=8 -vv -k "not test_kontrol_cse"'
           [ ${{ matrix.backend }} == 'legacy' ] && TEST_ARGS+=' --no-use-booster'
           docker exec --user github-user kontrol-ci-integration-${GITHUB_SHA} make cov-integration TEST_ARGS="${TEST_ARGS}"
       - name: 'Tear down Docker'
         if: always()
         run: |
           docker stop --time=0 kontrol-ci-integration-${GITHUB_SHA}
 
+  cse-tests:
+    needs: code-quality-checks
+    name: 'CSE Tests'
+    runs-on: [self-hosted, linux, normal, fast]
+    strategy:
+      fail-fast: false
+      matrix:
+        backend: ['booster']
+    timeout-minutes: 120
```

Comment:
Will this be enough time, given the timeout we saw previously?

Reply:
I think so, yes. If it's not, we will increase.